setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Yeah, I have one repo that automatically does this with Travis CI. Though I have largely switched to ReadTheDocs as it is just less effort to maintain.

ref: https://github.com/nanshe-org/nanshe/blob/9473837d3c8b978014c3f35c31c9f263f44d7549/.travis.yml#L87-L112